src/smokeTest/java/uk/gov/hmcts/reform/blobrouter/ReconciliationApiTest.java

```diff
@@ -0,0 +1,98 @@
+package uk.gov.hmcts.reform.blobrouter;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+
+import java.time.LocalDate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+import static org.springframework.http.HttpStatus.OK;
+import static org.springframework.http.HttpStatus.UNAUTHORIZED;
+
+public class ReconciliationApiTest {
+
+    private static final String RECONCILIATION_ENDPOINT_PATH = "/reform-scan/reconciliation-report/{date}";
+
+    private static Config config;
+    private static String apiGatewayUrl;
+    private static String validApiKey;
+
+    private static String statementsReport = "{\"report\": {\"envelopes\": []}}";
+
+    @BeforeAll
+    static void loadConfig() {
+        config = ConfigFactory.load();
+        apiGatewayUrl = getApiGatewayUrl();
+        validApiKey = getValidReconciliationApiKey();
+    }
+
+    @Test
+    void should_accept_request_with_valid_api_key() {
+        Response response = callReconciliationEndpoint(validApiKey).thenReturn();
+
+        assertThat(response.getStatusCode()).isEqualTo(OK.value());
+        assertThat(response.body().jsonPath().getString("id")).isNotEmpty();
+    }
```

Comment:
we should not post anything which ends up in success. smoke tests are executed on prod environment as well

Reply:
how can we test the valid scenario? is it possible?

Reply:
only through functional test. it is a POST endpoint - state changer in actual API. smoke test should not do it at any point in future